tensorflow/compiler/tf2xla/kernels/reduction_ops_common.cc

```diff
@@ -35,6 +35,12 @@ XlaReductionOp::XlaReductionOp(OpKernelConstruction* ctx,
       ctx, DataTypeToPrimitiveType(reduction_type_, &xla_reduction_type_));
 }
 
+// The default pre-processor directly returns the data. This can be overridden.
+xla::XlaOp XlaReductionOp::ProcessInput(xla::XlaBuilder* /*builder*/,
```

Comment:
The verb here needs to be more specific than `Process`.  `Preprocess` is fine.

Reply:
Updated.